### ISSUE TITLE
About file data storage

### ISSUE THREAD

**Issue by atiberghien** [pyofwave/PyOfWave#11]
hi, 
I have some questions about the file data storage (storage/files.py:
- In the interface definition what is the doc paramater,  a document object or a document URI ?
- What is the difference of a .ver and a .wave ? What about the hierarchical storage (wave -> wavelet -> blip -> document) ?
- What is the concept of Datasource successor ?

Thank's

Alban

**Comment by alcinnz** [comment]
There's some legacy left in there. When I was planning to use Google's Federation Protocol, I used the chain of responsibility (through the successor property) to drill down storage -> cache -> Federation. Now Wave is ONE protocol so that's unnecessary. 

The doc parameter is a document URI (drilling not implemented, wasn't needed in the legacy design). The document object has been replaced with `lxml`. 

The .ver (versions) file tracked all applied deltas (for Wave playback and restore) while .wave is the current version of that wave. 

Needs to be fixed up to work with `lxml` and the new architecture.
